### PR TITLE
feat: New macro to fetch data for ML models

### PR DESCRIPTION
**Pull request by denrou**

### CODE REVIEW

File: centreon/cloud/MLFETCH.mc2 [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    'interpolate' 'Boolean, if true interpolate missing value using a linear regression'
+  }
+} 'info' STORE
+
+<%
+  !$info INFO SAVE 'context' STORE
+
+  <%
+    // Organise parameters
+    // TODO refactoring
```
Comment (Nohzoh) [inline centreon/cloud/MLFETCH.mc2:57]: is this comment still valid?
if yes, I would prefer that we push clean code since the beginning :)

Comment (denrou) [reply centreon/cloud/MLFETCH.mc2:57]: It is still valid, but for now I haven't found a clean way to do it.

File: centreon/cloud/MLFETCH.mc2 [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  ] // Signature
+  'params' {
+    // Signature params description
+    'token' 'Read token to use to fetch the data.'
+    'class' 'Selector for the GTS classes. Can either be an exact match or a regular expression (the value should then start with a ~).'
+    'labels' 'Map of labels selectors. The key is the label name, the value a selector which can be an exact match or a regular expression (the value should then start with a ~).'
+    'end' 'Newest timestamp to consider when fetching datapoints. In platform time unit or ISO-8601.'
+    'start' 'Oldest timestamp to consider when fetching datapoints. In platform time unit or ISO-8601.'
+    'uniquelabels' 'List of label names to consider for grouping GTS with equivalence classes. If the list is empty all Geo Time Series™ will end up in the same equivalence class. If the labels parameter NULL, all labels will be considered.'
+    'diff' 'Boolean, if true compute compute the rate for each GTS of the LGTS'
```
Comment (Nohzoh) [inline centreon/cloud/MLFETCH.mc2:44]: there is a repetition
```suggestion
    'diff' 'Boolean, if true compute the rate for each GTS of the LGTS'
```
however, I am not sure to understand what this parameter stands for

File: centreon/cloud/MLFETCH.mc2 (hunk 1 above)
Comment (denrou) [reply centreon/cloud/MLFETCH.mc2]: As discussed, remove the TODO section and keep parameter definitions verbose

**Review by Nohzoh — APPROVED** [review]

**Pull request merged**